openedx/core/djangoapps/xblock/runtime/blockstore_runtime.py

```diff
@@ -67,9 +67,19 @@ def get_block(self, usage_id, for_parent=None):
                 # When a former XModule no longer needs to support the old runtime, its parse_xml_new_runtime method
                 # should be removed and its parse_xml() method should be simplified to just call the super().parse_xml()
                 # plus some minor additional lines of code as needed.
-                block = block_class.parse_xml_new_runtime(xml_node, runtime=self, keys=keys)
+                try:
+                    block = block_class.parse_xml_new_runtime(xml_node, runtime=self, keys=keys)
+                except Exception as exc:
+                    message = 'XBlockParseException when parse_xml_new_runtime xblock for usage_id {}'.format(usage_id)
```

Comment:
@s0b0lev the message is being passed to the exception, so it already contains the name of the exception, maybe we can remove it?

Reply:
Yeah, it makes sense. Thanks :)

Reply:
so aren't you removing it? :sweat_smile: I mean getting rid of `XBlockParseException` in the mssage